src/v2/Apps/PriceDatabase/Routes/MarketingLanding/Utils/formatCentsToDollars.ts

```diff
@@ -0,0 +1,17 @@
+export function formatCentsToDollars(value: number) {
+  if (value < 0) {
+    value = 0
+  }
+
+  const CENTS_IN_DOLLAR = 100
+
+  return new Intl.NumberFormat("en-US", {
+    // TODO: While artwork price insight stuff is all currently calculated
+    // in dollars, we might need to return to this later.
+    currency: "USD",
+
+    style: "currency",
+  })
+    .format(Math.round(value / CENTS_IN_DOLLAR))
+    .replace(".00", "")
+}
```

Comment:
If the functionality is similar, can we consolidate all of our `formatCentsToDollars` helpers under a single `v2/Utils/formatCentsToDollars`? 

<img width="403" alt="Screen Shot 2021-08-02 at 9 22 42 AM" src="https://user-images.githubusercontent.com/236943/127893310-5906734e-b2a3-4f02-81f2-c99172190d8c.png">

Reply:
👍